server/controllers/homepageController.ts

```diff
@@ -78,7 +76,7 @@ export default class HomepageController {
     }
   }
 
-  async getServices(req: CustomRequest, res: Response, next: NextFunction) {
+  public async getServices(req: Request, res: Response, next: NextFunction) {
```

Comment:
You're passing in `req`, `res` and `next` but only using `res` for `res.locals.user` and `res.locals.clientToken`
Would make more sense to just pass in those 2 objects.